api/client/samples/anomaly_detection/sample_anomaly_detection.ipynb

```diff
@@ -0,0 +1,799 @@
+{
```

Comment:
<p>The below may be a bit clearer as to what the notebook does. Perhaps, even provide an example to help visualize what the notebook can do.</p><p>In this notebook, we outline a process for <strong>identifying </strong>anomalies <strong>in</strong> geospatial data. <strong>For example, this notebook can be used to set up alerts for when the amount of rainfall is higher than usual in a certain area, or if the temperature is colder this month than it has been historically, etc...&nbsp;</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>nice i edited a bit and used your modifications!</p>